setup-ros.sh

```diff
@@ -25,17 +27,17 @@ apt-get install --no-install-recommends --quiet --yes tzdata
 apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 \
     --recv-keys C1CF6E31E6BADE8868B172B4F42ED6FBAB17C654
 
-echo "deb http://packages.ros.org/ros/ubuntu $(lsb_release -sc) main" \
+echo "deb http://packages.ros.org/ros${ROS_REPO_SUFFIX}/ubuntu $(lsb_release -sc) main" \
```

Comment:
Why not pass the URL directly setting half of the string seems fairly arbitrary?

i.e.

```
ROS_APT_HTTP_REPO_URL=http://packages.ros.org/ros-testing/
```

Reply:
might be a list like `ROS_APT_REPO_URLS="http://packages.ros.org/ros http://packages.ros.org/ros2-testing"` ? Most of the images we build probably only need one list entry anyway, no need to add both ros1 and ros2 to all images

Reply:
Done.